docs/_content/ontwikkelaars/aan-de-slag.md

```diff
@@ -266,7 +266,7 @@ Autorisatiecomponent (AC)
    Don't forget to enter your own secret (bottom right) instead of `your-256-bit-secret`!
    
    _Creating a JWT does not register the secret with the hosted reference components. 
-   See the [API guides](./guides) how this works._
+   See the [API guides](https://ref.tst.vng.cloud/ontwikkelaars/guides) how this works._
```

Comment:
This relative URL should work, why is this change needed? It's a known issue that the pure markdown doc links are broken, but it's either published doc urls working or markdown docs in github. So we stick to functional published docs.

Absolute URLs including the domain are not convenient, since we're migrating to different domains.

Reply:
You're right. I was testing these links on GitHub (where the relative URLs don't work), but they are made for https://ref.tst.vng.cloud/, where they do work. I'll revert the changes to the URLs.